src/main/scala/cromwell/binding/types/WdlStringType.scala

```diff
@@ -21,6 +21,7 @@ case object WdlStringType extends WdlPrimitiveType {
 
   override def add(rhs: WdlType): Try[WdlType] = rhs match {
     case WdlStringType | WdlIntegerType | WdlFloatType => Success(WdlStringType)
+    case WdlFileType => Success(WdlFileType)
```

Comment:
Yeah, I suppose String + File probably make the most sense as evaluating to a String... I can change